ext/standard/tests/file/bug39551.phpt

```diff
@@ -4,6 +4,7 @@ Bug #39551 (Segfault with stream_bucket_new in user filter)
 <?php
 
 $bucket = stream_bucket_new(fopen('php://temp', 'w+'), '');
+var_dump($bucket);
```

Comment:
As you're really only interested in the class, perhaps use:

```
echo get_class($bucket), "\n";
```

Then you don't have to update the test later when `bucket` becomes an object, or `datalen` becomes `dataLength`.

Reply:
It would be good idea if there were other tests which ensured the structure of `StreamBucket`. But I'd prefer to keep this test as-is in this case, since that's the only place where the properties are displayed.